### PR TITLE
Update less-than-one-year.md

### PR DESCRIPTION
**Pull request by mollyblake**
- removed pivotal numbers
- changed format to subway map 
- removed extraneous text
- fixed links

### CODE REVIEW

File: _employment/job-seekers/less-than-one-year.md [hunk 1]
```diff
@@ -11,43 +11,81 @@ template: 1-topic-landing
...
 - Contact your local Transition Assistance Program (TAP) manager to schedule the mandatory Transition Goals, Plans, Success (GPS) curriculum. If you are unable to attend this training, it is available through [Joint Knowledge Online](https://jkodirect.jten.mil). You must complete the VA Benefits Briefings I and II, the Department of Labor Employment Workshop, and an Individual Transition Plan (ITP). ITPs are service specific, so check [documents](https://dodtap.mil/index.html) related to your branch. 
-- Review [military transcripts pivotal#110298578](/_dummy-placeholder.html) and service records for accuracy. 
-- Think about which industries and careers interest you, using [tools#110298582](/_dummy-placeholder.html) such as o*Net and CareerScope. 
-- Start developing a résumé by [translating military skills 110298580](/_dummy-placeholder.html) into equivalent civilian skills. Import these using the [Résumé Builder](/_dummy-placeholder.html). 
-- Clean up your personal social-media footprint and establish a presence on professional sites like LinkedIn. 
+- Review [military transcripts](employment/job-seekers/military-transcrips) and service records for accuracy.
```
Comment (emilyville) [inline _employment/job-seekers/less-than-one-year.md]: missing t in transcripts !

**Comment by emilyville** [comment]
These are the links making this fail - other than the one with the typo above I can add the to the"safe list" @ayaleloehr created and then this will pass.

- ./_site/employment/job-seekers/less-than-one-year/index.html
  *  internally linking to employment/job-seekers/federal-employment, which does not exist (line 210)
  *  internally linking to employment/job-seekers/interest-profiler, which does not exist (line 187)
  *  internally linking to employment/job-seekers/military-skills-translator, which does not exist (line 188)
  *  internally linking to employment/job-seekers/military-transcrips, which does not exist (line 186)
  *  internally linking to employment/job-seekers/resume-builder, which does not exist (line 188)

File: _employment/job-seekers/less-than-one-year.md (hunk 1 above)
Comment (mollyblake) [inline _employment/job-seekers/less-than-one-year.md]: @emilyville when I go back into github to fix this missing T, none of the changes I made are reflected. So should I wait a bit for it to reflect the changes I made?

Comment (emilyville) [inline _employment/job-seekers/less-than-one-year.md]: nah, it should be instant. I can fix it now.

**Comment by mollyblake** [comment]
Should be a subway map!

**Comment by emilyville** [comment]
subway map looks a little broken, I'll fix that too 

screencapture-vets-website-pr-1526-herokuapp-com-employment-job-seekers-less-than-one-year-1458942197614

**Comment by emilyville** [comment]
Actually, all of these are just missing the beginning / "/link/link"

File: _employment/job-seekers/less-than-one-year.md [hunk 2]
```diff
@@ -11,43 +11,81 @@ template: 1-topic-landing
...
 - Contact your local Transition Assistance Program (TAP) manager to schedule the mandatory Transition Goals, Plans, Success (GPS) curriculum. If you are unable to attend this training, it is available through [Joint Knowledge Online](https://jkodirect.jten.mil). You must complete the VA Benefits Briefings I and II, the Department of Labor Employment Workshop, and an Individual Transition Plan (ITP). ITPs are service specific, so check [documents](https://dodtap.mil/index.html) related to your branch. 
-- Review [military transcripts pivotal#110298578](/_dummy-placeholder.html) and service records for accuracy. 
-- Think about which industries and careers interest you, using [tools#110298582](/_dummy-placeholder.html) such as o*Net and CareerScope. 
-- Start developing a résumé by [translating military skills 110298580](/_dummy-placeholder.html) into equivalent civilian skills. Import these using the [Résumé Builder](/_dummy-placeholder.html). 
-- Clean up your personal social-media footprint and establish a presence on professional sites like LinkedIn. 
+- Review [military transcripts](employment/job-seekers/military-transcripts) and service records for accuracy.
```
Comment (emilyville) [inline _employment/job-seekers/less-than-one-year.md]: We're missing the beginning / on this link

**Comment by emilyville** [comment]
k, this page should be all set now! Assigning to @webinista for a review. 
(The copy has been reviewed, so this can just be markup and/or viewing)

https://vets-website-pr-1526.herokuapp.com/employment/job-seekers/less-than-one-year/

**Pull request merged**